deebot_client/commands/json/map.py

```diff
@@ -22,6 +24,14 @@ class GetCachedMapInfo(JsonCommandWithMessageHandling, MessageBodyDataDict):
     """Get cached map info command."""
 
     name = "getCachedMapInfo"
+    # version definition for using type of getMapSet v1 or v2
+    get_map_set_version: int = 1
+
+    def __init__(
+        self, args: dict[str, Any] | list[Any] | None = None, version: int = 1
+    ) -> None:
+        self.get_map_set_version = version
```

Comment:
```suggestion
        match (version):
            case 1:
                self._map_set_command = GetMapSet
            case 2:
                self._map_set_command = GetMapSetV2
            case _:
                raise ValueError(f"version={version} is not supported")
```

We can directly store the type of the get command

Reply:
updated to use the match-case option